src/App.tsx

```diff
@@ -29,11 +29,7 @@ function App() {
 
   useEffect(() => {
     hospitalInfoService.getHospitalInfo().then((res: HospitalInfo[]) => {
-      const closedHospitals = res.filter(
-        (hospital) => hospital.status === "Closed"
-      );
-      console.log("Closed Hospitals:", closedHospitals);
-      setHospitals(closedHospitals);
+      setHospitals(res);
     });
     generalInfoService.getGeneralInfo().then((res) => console.log(res));
```

Comment:
Do we need these?  If so, add a comment why.

Reply:
We need this to display 'Hospitals actively funding' in green and 'closed hospitals' in tomato color. If we don't do it this way, we would have to create separate markers for green and tomato colors, which would reduce reusability.

Reply:
Sorry, the line reference is for lines 34-35